lib/minitest/reporters/base_reporter.rb

```diff
@@ -85,12 +85,13 @@ def print(*args)
         io.print(*args)
       end
 
-      def print_info(e, name=true)
-        print "#{e.exception.class.to_s}: " if name
+      def print_info(e, error=true)
         e.message.each_line { |line| print_with_info_padding(line) }
 
-        trace = filter_backtrace(e.backtrace)
-        trace.each { |line| print_with_info_padding(line) }
+        unless error
+          trace = filter_backtrace(e.backtrace)
+          trace.each { |line| print_with_info_padding(line) }
+        end
```

Comment:
@os97673 Have updated the code.. it should be clear now why the change is being made.